### ISSUE TITLE
Inserter search focus style and alignments

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#1378]
Splitting this out from #1187 that is going to move the inserter search at the top. I'd like to propose two small improvements:

![screen shot 2017-06-22 at 17 32 08](https://user-images.githubusercontent.com/1682452/27442982-103d9c0e-5772-11e7-9d14-8b332c766798.png)

- accessibility: the search field should have a focus style
- readability/design: it would be nice to have the search placeholder/text, the block groups title, and the block icons horizontally aligned; currently, they all have a different left margin/padding which makes visually scanning the content a bit awkard

**Comment by jwold** [comment]
@afercia 

> accessibility: the search field should have a focus style

Do you mean something like adding a blue outline? 

<img width="964" alt="blue outline" src="https://user-images.githubusercontent.com/3411173/27564150-ad5accda-5a8a-11e7-8032-73b67fc66100.png">

> currently, they all have a different left margin/padding which makes visually scanning the content a bit awkward

Not quite following, can you clarify what you mean?

**Comment by afercia** [comment]
@jwold re: focus style: yes. as it's standard for all form fields in WordPress. Re: left alignment, see screenshots on the PR #1393
